.github/workflows/lint.yml

```diff
@@ -36,8 +36,8 @@ jobs:
           cargo test --release --manifest-path src/quacs-rs/Cargo.toml
           echo "Running wasm-pack test --node"
           time wasm-pack test --release --node src/quacs-rs
-          echo "Running wasm-pack test --node -- --all-features"
-          time wasm-pack test --release --node src/quacs-rs -- --all-features > /dev/null
+          echo "Running wasm-pack test --node --"
+          time wasm-pack test --release --node src/quacs-rs -- > /dev/null
```

Comment:
I don't think the `--` at the end is required.

Reply:
Good catch, This seemed to make the two lines above it identical so I just removed the whole thing.